lib/ramble/ramble/pipeline.py

```diff
@@ -597,11 +604,26 @@ def _execute(self):
                 f.write(syaml.dump_config(repo_data))
 
         repo_path = os.path.join(self.workspace.named_deployment, self.object_repo_name)
-        object_types = ["applications", "modifiers", "packages"]
+        object_types = [
+            "applications",
+            "modifiers",
+            "packages",
+            "package_managers",
+            "base_applications",
+            "base_modifiers",
+            "base_package_managers",
+        ]
         for object_type in object_types:
             fs.mkdirp(os.path.join(repo_path, object_type))
 
-        for conf_file in ["repo.yaml", "modifier_repo.yaml"]:
+        for conf_file in [
+            "repo.yaml",
+            "modifier_repo.yaml",
+            "package_manager_repo.yaml",
+            "base_application_repo.yaml",
+            "base_modifier_repo.yaml",
+            "base_package_manager_repo.yaml",
+        ]:
             with open(os.path.join(repo_path, conf_file), "w+") as f:
                 f.write("repo:\n")
```

Comment:
If we are writing different config files for each (we don't need to) then the top level attribute should be the same as the config section name (i.e. filename sans `'.yaml'`)

Reply:
I see, changed to only write out the unified config.